src/main/scala/subsystem/FrontBus.scala

```diff
@@ -18,14 +18,15 @@ case object FrontBusKey extends Field[FrontBusParams]
 
 class FrontBus(params: FrontBusParams)
               (implicit p: Parameters) extends TLBusWrapper(params, "front_bus")
-    with HasTLXbarPhy
-    with HasCrossing {
-  val crossing = params.sbusCrossing
+    with HasTLXbarPhy {
+
+  protected val sbus_xing = new CrossingHelper(this, params.sbusCrossing)
+  def crossTLOut(implicit p: Parameters): TLNode  = sbus_xing.crossTLOut
```

Comment:
I am not sure I approve of these forwarding functions. I think it might be more clear to have the caller explicitly refer to the crossing by name. eg: fbus.sbus_xing.crossTLOut(...)

Reply:
I think I agree